meta-lmp-bsp/recipes-bsp/u-boot/u-boot-base-scr/apalis-imx8/boot.cmd

```diff
@@ -1,3 +1,14 @@
+setenv fdt_file ${fdtfile}
+echo "Using freescale_${fdt_file}"
+
+# Default boot type and device
+setenv devtype mmc
+setenv devnum 0
+setenv boot_file Image
+setenv bootfile ${boot_file}
+setenv kernel_addr_r 0x80280000
```

Comment:
any reason to re-define kernel_addr_r, if it's already in `include/configs/apalis-imx8.h`?

Reply:
Let me check some if these synchronization changes.

This probably isn't needed.

Reply:
Dropped the addition of the top of this file.  Distro boot is enabled and this all should work out of the box.